### DISCUSSION TITLE
Island Creation Spawnpoint Offset

### DISCUSSION THREAD

**Discussion by StinkyPinky045** [BG-Software-LLC/SuperiorSkyblock2#2130] [Pending Feature Requests]
Hi!
i would love to get this feature implemented where we will be able to move Island spawnpoint without players having to do ./island setspawn.

If island size is set to chunkgrid ex. 16. 32. 64. 128. 512. 1024.
island spawnpoint will always be chunk position 0 ~ 0 on island creation
![XPkX5Dq](https://github.com/BG-Software-LLC/SuperiorSkyblock2/assets/141526031/dc039b97-8398-4a60-9a08-47a03f10b843)
i know we can change spawnpoint with /is admin schematic, but that would move the island and not the player
but i would love to be able to change Player offset without changing island size
![QZapiHO](https://github.com/BG-Software-LLC/SuperiorSkyblock2/assets/141526031/860c09c1-79ad-4c81-80d5-ede770112c6e)

**Comment by OmerBenGera** [comment]
Duplicate of #2126
